queries/rbs/highlights.scm

```diff
@@ -54,9 +54,24 @@
   (method_name
     [
       (identifier)
+      (identifier_suffix)
       (constant)
+      (constant_suffix)
       (operator)
       (setter)
+      (constant_setter)
+    ] @function.method))
+
+(attribute_member
```

Comment:
can these be combined via alternations (`[method_member attribute member]`)?

Reply:
I tried to combine these like the following.

```
(([(method_member) (attribute_member)]
  (method_name
    [
     (identifier)
     (identifier_suffix)
     (constant)
     (constant_suffix)
     (operator)
     (setter)
     (constant_setter)
    ] @method)))
```

But tree-sitter cannot highlight properly.
Is there a mistake?

Reply:
Yeah, that has a different (incorrect) meaning (missing nesting). Nevermind then, thanks for checking!